nvm.sh

```diff
@@ -726,8 +724,7 @@ nvm_resolve_alias() {
       break
     fi
 
-    if [ -n "${ALIAS_TEMP}" ] \
-      && command printf "${SEEN_ALIASES}" | nvm_grep -e "^${ALIAS_TEMP}$" > /dev/null; then
+    if command printf "${SEEN_ALIASES}" | nvm_grep -e "^${ALIAS_TEMP}$" > /dev/null; then
```

Comment:
Can you explain this change? If `ALIAS_TEMP` is empty, we shouldn't need to run the printf or grep.